src/auth/auth.controller.ts

```diff
@@ -74,10 +74,11 @@ export class AuthController extends BaseController implements IAuthController {
 	}
 
 	async activate(req: Request, res: Response, next: NextFunction): Promise<void> {
-		const { token: activationToken } = req.query;
+		const user = req.user;
+		console.log(user);
```

Comment:
remove console.log please
using debugging mode is more convenient

Reply:
done